login_utils.py

```diff
@@ -7,7 +7,7 @@
 def login_verification():
     pass_text = request.form['pass']
     username_text = request.form['name']
-    return db_management.does_user_exist(username_text, pass_text)
+    return db_management.check_if_user_exists(username_text, pass_text)
```

Comment:
I think it's simpler to use this line instead: 
```suggestion
    return db_management.is_user_exists(username)
```

Reply:
changed the name - but remember this method serves for logins. it needs both the password and username

Reply:
No - again, this function checks if a user exists. Why do you need the password? you don't use it

Reply:
i belive the name was simply misleading. changed it accordingly. 
the method is to check if there is a row in our sql table which has both a username and a password,
 matching with our input. changed the name to "check_login_parameters"